### ISSUE TITLE
check-revoke in config ignored

### ISSUE THREAD

**Issue by ImNtReal** [nabijaczleweli/cargo-update#239]
My config.toml includes:
```
[http]
check-revoke = false
```

Running `cargo install` works for me, but if I try to run cargo-update, I get a failure:
```
 jmpugh@408itslt026  ~  .cargo  cargo install-update --all
    Polling registry 'https://index.crates.io/'
Failed to update index repository crates-io: package blindfold: [35] SSL connect error (schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.).
```

I'm not sure what else you need to know for this. I'm on Windows 10. I can tell it's reading the config file because it warns of bad options, but seems to ignore this one option that I need because I'm on a corporate network that does TLS inspection.

**Comment by nabijaczleweli** [comment]
Nope, we just don't check this table at all yet. If you use any of the other keys under `[http]`, then now would be a good time to share that fact.

Please try the current `master` branch (at least d225a42ef4a76e36d31ba6abd7bc8ebd09d4a31b).

**Comment by ImNtReal** [comment]
The only others I have under [http] is cainfo and debug.

**Comment by nabijaczleweli** [comment]
`cainfo` forwarded in 92b0ae2053319dcab5ce40f01bec5d308f073563

**Comment by nabijaczleweli** [comment]
Released in [v13.1.0](https://github.com/nabijaczleweli/cargo-update/releases/tag/v13.1.0).
